src/pug/alpakka/components.pug

```diff
@@ -105,9 +106,12 @@ block page
       table.data
         tr
           td USB to TTL serial cable
-          td: +ax('https://www.adafruit.com/product/954') Visit website
+          td: +ax('https://www.adafruit.com/product/954') Adafruit shop
         tr
           td JST PH 4-pin cable
-          td: +ax('https://www.adafruit.com/product/3955') Visit website
-
+          td: +ax('https://www.adafruit.com/product/3955') Adafruit shop
 
+    div.box
+      h2 About sourcing from Input Labs
+      p By getting some of the components and materials directly from #[+ax('https://inputlabs.squarespace.com') Input Labs Distribution], you help us to continue doing the research we do, and to keep releasing awesome open-source and accessible controllers for everybody.
+      p Also it increases your charisma by 10 points.
```

Comment:
Either "Also, it increases" or better "It also increases" (or drop it altogether)